src/lib/store.js

```diff
@@ -22,7 +22,7 @@ const preloadFiles = (files) => {
     entries: _get(_files, 'entries', [])
       .map((file) => {
         let hasSize;
```

Comment:
minor: more compact and less undefined
```suggestion
        let hasSize = file.size >= 0;
```